outline

```diff
@@ -185,10 +188,6 @@ Fuse : articles/index.md
 		Starting an Android Activity : articles/native-interop/android-start-activity.md
 		iOS Capabilities : articles/native-interop/ios-capabilities.md
 
-	Alive UX Kit : articles/alive/alive.md
-		Components : api-docs/api/alive.json
-		Themes : articles/alive/themes.md
-
```

Comment:
How did you update this file? By hand, or is there some automatic way?

Reply:
I did it by hand, mostly cutting and pasting though to retain previous formatting. Only real new things were around Observables, Models API and sound.

Reply:
OK, thanks :)

Reply:
Oh I did reduce the one redundant part, which was the "Full Reference" link and sub links.